web/controllers/caseload_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Healthlocker.CaseloadController do
+  use Healthlocker.Web, :controller
+
+  alias Healthlocker.{EPJSTeamMember, EPJSUser, ReadOnlyRepo, User}
+
+  def index(conn, _params) do
```

Comment:
yeah, I was thinking it may be good to have the clinician_id for each connected user to possibly help with that. I didn't want to spend too much time on this particular part since it's temp and will likely be done quite different with the real data, and should still be quick enough with the amount of seed data we have.